docs/install.md

```diff
@@ -1,7 +1,7 @@
 
 # Install with Docker
 
-The simplest way to get the entire platform up an running from scratch is using docker. Most of the servers are auto-started together with docker-compose but the keycloak container must be started and configured on its own first. The instructions below set up the platform with HTTP and then use a reverse proxy to pick up the ports. This way you can omit the proxy step and run the platform with http when initialling setting up and testing.
+Since PORI is a production-ready, instituion-level, scaleable platform, the simplest way to get the entire platform up an running from scratch is using docker. Most of the servers are auto-started together with docker-compose but the keycloak container must be started and configured on its own first. The instructions below set up the platform with HTTPS and then use a reverse proxy to pick up the ports. This way you can omit the proxy step and run the platform with http when initialling setting up and testing.
```

Comment:
instituion -> institution; up an running -> up and running

Reply:
fixed